src/QirRuntime/README.md

```diff
@@ -32,11 +32,9 @@ CI builds and tests are enabled for this project. The build has no external depe
 
 1. Install Clang, Ninja and CMake from the public distros.
 2. Add all three to your/system `%PATH%`.
-3. Install VS 2019 (Clang uses MSVC's standard library on Windows).
-4. Install .NET Core 3.1
-5. <_optional_> Either clone LLVM's repo and build LLVM locally to produce your own `opt.exe` and other IR tools or download LLVM's dev package.
-6. Install clang-tidy and clang-format if your Clang/LLVM packages didn't include the tools.
-7. <_optional_> To use build/test scripts install Python 3.8.
+3. Install VS 2019 and enable "Desktop development with C++" component (Clang uses MSVC's standard library on Windows).
+4. Install clang-tidy and clang-format if your Clang/LLVM packages didn't include the tools.
+5. <_optional_> To use build/test scripts install Python 3.8.
```

Comment:
Is Python still required? looks like you moved everything to PS1...

Reply:
python script gives me more flexibility with local builds (e.g. just running make part of cmake) and it also creates windows/linux subfolders so WSL builds don't override the host (which is unnecessary in the cloud build). I'd like to keep it, at least for now.